unrpyc.py

```diff
@@ -320,4 +390,4 @@ def rpyc_check(inp):
         print("Decompilation of %d file%s successful, but decompilation of %d file%s failed" % (good, 's' if good>1 else '', bad, 's' if bad>1 else ''))
 
 if __name__ == '__main__':
-    main()
+    main(_parse_args())
```

Comment:
```python
if __name__ == '__main__':
    main()
```

Is an established coding style, I'd rather call _parse_args() as the first thing in main()